docs/technical-design/database-diagram.md

```diff
@@ -0,0 +1,110 @@
+# Diagram of the db with future designs
+
+## Current DB
+
+```mermaid
+erDiagram
+
+HealthPlanPackageTable ||--|{ HealthPlanRevisionTable: "has many"
+HealthPlanPackageTable }|--|| State: "has many"
+
+HealthPlanPackageTable {
+    string id
+    string stateCode
+}
+
+HealthPlanRevisionTable {
+    string id
+    string pkgID
+    datetime createdAt
+    bytes   formDataProto
+    datetime submittedAt
+    string submittedBy
+    string submittedReason
+    datetime unlockedAt
+    string unlockedBy
+    string unlockedReason
+}
+
+State {
+    string stateCode
+    string name
+    int latestSubmissionNumber
+}
+
+User {
+    int id
+    string givenName
+    string familyName
+    string email
+    string role
+}
+
+User }|--|{ State: "many to many"
+```
+
+## QA Doc Upload Design
+
+```mermaid
+
+erDiagram
+
+HealthPlanPackageTable ||--|{ HealthPlanRevisionTable: "has many"
+HealthPlanPackageTable }|--|| State: "has many"
+
+HealthPlanPackageTable {
+    string id
+    string stateCode
+}
+
+HealthPlanRevisionTable {
+    string id
+    string pkgID
+    datetime createdAt
+    bytes   formDataProto
+    datetime submittedAt
+    string submittedBy
+    string submittedReason
+    datetime unlockedAt
+    string unlockedBy
+    string unlockedReason
+}
+
+State {
+    string stateCode
+    string name
+    int latestSubmissionNumber
+}
+
+User {
+    int id
+    string givenName
+    string familyName
+    string email
+    string role
+}
+
+User }|--|{ State: "many to many"
+
+QuestionRound {
+    string id
+    string pkgID
+    string name
+    string requestedBy  
+    date requestDate
+    date responseDate
+}
+
+QuestionRound ||--|| User: "requestedBy"            
+
+QuestionRound ||--|{ Document: "has many"
+HealthPlanPackageTable ||--|{ QuestionRound: "has many"
+
+Document {
```

Comment:
Here's a version of DocumentsTable from a couple of RSP ago

```
id 
name 
s3Url
documentCategory
createdAt
sha
virusScan
```

Reply:
I don’t think we need the category or sha for now, those were both considerations for pulling documents out of the proto, I think